pkg/domain/infra/abi/images.go

```diff
@@ -452,7 +452,8 @@ func (ir *ImageEngine) Load(ctx context.Context, opts entities.ImageLoadOptions)
 	if !opts.Quiet {
 		writer = os.Stderr
 	}
-	name, err := ir.Libpod.LoadImage(ctx, opts.Name, opts.Input, writer, opts.SignaturePolicy)
+	imgName := opts.Name + ":" + opts.Tag
+	name, err := ir.Libpod.LoadImage(ctx, imgName, opts.Input, writer, opts.SignaturePolicy)
```

Comment:
Hum. Paging  @mheon @jwhonce 
- ⚠️  *What’s the semantics of `ImageLoadOptions`*? We must establish that and document that. (See also https://github.com/containers/podman/pull/6811#discussion_r473009515 )
- Why is it safe to change this behavior now?
- `pkg/domain/infra/tunnel.ImageEngine.Load` uses `Name`+`Tag` in yet another different way. Which one is correct?